### DISCUSSION TITLE
Install & Upgrade guidance for Release notes

### DISCUSSION THREAD

**Discussion by asoorm** [TykTechnologies/tyk-operator#336] [Ideas]
In the release notes for our operator - we currently have a section called **Docker images** which I am not certain if it is that useful.

<img width="469" alt="Screenshot 2021-07-22 at 10 46 00" src="https://user-images.githubusercontent.com/1465130/126620071-85f0fc1a-54e5-4890-b5db-32ae3d419671.png">

I think that we can deprecate this section, and replace it with 2 headings moving forward:

* Install
* Upgrade

Largely, these sections should be the same for every release, however the Upgrade notes will also identify breaking changes.

**Comment by gernest** [comment]
makes sense. Since from `v0.8.0` we will be using the helm repository for installation, the section willl be any chages that are helm chart specific.
